web-playground/src/components/editor.tsx

```diff
@@ -58,6 +58,8 @@ function toMode(language: string) {
         case "nickel":
             // Missing highlighting for Nickel, but we know.
             return "plain_text";
+        case "cfml":
+            return "cfml";
```

Comment:
Again, because of WASM and the playground are currently in maintenance mode, I would recommend removing this. It won't be used in the foreseeable future and its existence here implies that the feature is available and tested, which it isn't. i.e., Its presence is misleading.